tasks/cops_documentation.rake

```diff
@@ -12,6 +12,8 @@ end
 
 desc 'Generate docs of all cops departments'
 task generate_cops_documentation: :yard_for_generate_documentation do
+  RuboCop::ConfigLoader.inject_defaults!("#{__dir__}/../config/default.yml")
```

Comment:
How come this line is necessary?

Reply:
It may eventually be removed, but it is still needed for merging the `config/default.yml` configuration used for documentation generation.

From a technical perspective, `Plugin` is called back from RuboCop core during runtime and test execution. However, documentation generation has not yet been integrated with this plugin callback mechanism. In any case, this logic is only required for the documentation generation process, while runtime works with plugins.

Reply:
Thanks